leetcode/README.md

```diff
@@ -1,100 +1,79 @@
-| 977  | [Squares of a Sorted Array](https://leetcode.com/problems/squares-of-a-sorted-array/)                                           | [C](./src/977.c)  | Easy       |
-| 1008 | [Construct Binary Search Tree from Preorder Traversal](https://leetcode.com/problems/construct-binary-search-tree-from-preorder-traversal/description/) | [C](./src/1008.c) | Medium |
-| 1019 | [Next Greater Node In Linked List](https://leetcode.com/problems/next-greater-node-in-linked-list/)                             | [C](./src/1019.c) | Medium     |
-| 1089 | [Duplicate Zeros](https://leetcode.com/problems/duplicate-zeros/)                                                               | [C](./src/1089.c) | Easy       |
-| 1184 | [Distance Between Bus Stops](https://leetcode.com/problems/distance-between-bus-stops/)                                         | [C](./src/1184.c) | Easy       |
-| 1189 | [Maximum Number of Balloons](https://leetcode.com/problems/maximum-number-of-balloons/)                                         | [C](./src/1189.c) | Easy       |
-| 1207 | [Unique Number of Occurrences](https://leetcode.com/problems/unique-number-of-occurrences/)                                     | [C](./src/1207.c) | Easy       |
-| 2130 | [Maximum Twin Sum of a Linked List](https://leetcode.com/problems/maximum-twin-sum-of-a-linked-list/)                           | [C](./src/2130.c) | Medium     |
+# 📚 Contributing 📚
+
+We're glad you're interested in adding C LeetCode solutions to the repository.\
+Here we'll be explaining how to contribute to LeetCode solutions properly.
+
+## 💻 Cloning/setting up the project 💻
+
+First off, you'll need to fork the repository [**here**](https://github.com/TheAlgorithms/C/fork).\
+Then, you'll need to clone the repository to your local machine.
+
+```bash
+git clone https://github.com/your-username/C.git
+```
+
+After that, you'll need to create a new branch for your solution.
+
+```bash
+git checkout -b solution/your-solution-name
+```
+
+## 📝 Adding a new solution 📝
+
+All LeetCode problems can be found [**here**](https://leetcode.com/problemset/all/).\
+If you have a solution to any of these problems (which are not being [**repeated**](https://github.com/TheAlgorithms/C/blob/master/leetcode/DIRECTORY.md)), that's great! Here are the steps:
+
+1. Add a new file in `leetcode/src` with the number of the problem.\
+    - For example: if the problem's number is 98, the filename should be `98.c`.
+2. Provide a small description of the solution at the top of the file. A function should go below that. For example:
+
+```c
+/**
+ * Return an array of arrays of size *returnSize.
+ * The sizes of the arrays are returned as *returnColumnSizes array.
+ * Note: Both returned array and *columnSizes array must be malloced, assume caller calls free().
+ */
+```
+
+3. Do not provide a `main` function. Use the required standalone functions instead.
```

Comment:
Non-blocking; I really think that we should check the base cases that are provided on the website. I can create an action testing framework that could check this dynamically if this is something that we want 😄

Reply:
That sounds like an awesome idea! Thank you.